scripts/observers/config.py

```diff
@@ -0,0 +1,113 @@
+import os
+import toml
+import sys
+import shutil
+from pathlib import Path
+from git import Repo
+from dotenv import load_dotenv
+
+load_dotenv()
+
+observer_dir = str(os.getenv('OBSERVER_DIR'))
+METACHAIN = 4294967295
+
+
+def update_toml_indexer(path, shard_id):
+    # prefs.toml
+    data1 = toml.load(path+"/prefs.toml")
+    data1['config']['web-socket']['server-url'] = str(shard_id)
+    if shard_id != METACHAIN:
+        data1['config']['web-socket']['server-url'] = "localhost:" + str(22111+shard_id)
```

Comment:
Is `22111` the port base? Perhaps define a constant or receive it in the env file?

Reply:
added  a constant